HHS/Head-Start-TTADP
# [Production] Automatically set goal source for curated goals, frontend template for session objectives

## Description of change
For curated goals (monitoring and FEI):

- Historical goals will have their source automatically set
- Any new curated goals will also have their source automatically set, and it will be read-only

This PR also includes the template for the session objective. This is a front end only change and will not cause anything in production to change.

## How to test
Confirm that FEI goals are present in the RTR for curated goals, and that if you create a new goal from a curated template the source is present and read-only.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2163
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2342

## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
